consensus/ssz_types/src/variable_list.rs

```diff
@@ -320,7 +320,8 @@ mod test {
 
         let vec = vec![];
         let fixed: VariableList<u64, U4> = VariableList::from(vec);
-        assert_eq!(&fixed[..], &vec![][..]);
+        let expected: Vec<u64> = vec![];
```

Comment:
Once I added the new modules, I was getting this error:

![Screen Shot 2020-10-26 at 12 41 00 PM](https://user-images.githubusercontent.com/5160426/97201272-a23f9880-1788-11eb-9a59-3feaf3f6759e.png)

Which is why I made this update. But I don't know why adding the new modules would cause it to pop up...

Reply:
Can you use `&[]` instead of `&vec![][..]`?

Reply:
It is weird that the modules caused the change tho.

Reply:
updated 👍